### PR TITLE
#878 more Text classes refactored to use TextEnvelope

### PR DESCRIPTION
**Pull request by Vatavuk**
Refactored some `Text` classes to use `TextEnvelope` for #878 
I had to suppres pqulice warnings in three classes. I don't know why qulice is complaining, must probably a bug.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#880](https://github.com/yegor256/cactoos/issues/880) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#880](https://github.com/yegor256/cactoos/issues/880) is assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), here is [why](https://www.0crat.com/footprint/C63314D6Z/1372041762); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @llorllale[/z](https://www.0crat.com/u/llorllale) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer

File: src/main/java/org/cactoos/text/PaddedStartText.java [hunk 1]
```diff
@@ -57,21 +43,16 @@
...
+        super((Scalar<String>) () -> {
+            final String original = text.asString();
+            final int diff = length - original.length();
+            final StringBuilder builder = new StringBuilder();
+            for (int len = diff; len > 0; --len) {
+                builder.append(symbol);
+            }
+            builder.append(original);
+            return builder.toString();
+        });
```
Comment (paulodamaso) [inline src/main/java/org/cactoos/text/PaddedStartText.java]: @Vatavuk WDYT about using `RepeatedText` to generate this pad string?

Comment (Vatavuk) [reply src/main/java/org/cactoos/text/PaddedStartText.java]: @paulodamaso good idea! Refactored it in 2146d91

**Comment by llorllale** [comment]
@rultor merge

**Comment by 0crat** [comment]
@elenavolokhova[/z](https://www.0crat.com/u/elenavolokhova) please review this job completed by @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#880](https://github.com/yegor256/cactoos/issues/880) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by elenavolokhova** [comment]
@paulodamaso According to our Policy:
>The code reviewer found at least three problems in the code

Only one issue was found during this review. 
Please, confirm that you'll try to find at least 3 issues next time.

**Comment by paulodamaso** [comment]
@elenavolokhova Confirmed, thanks.

**Comment by elenavolokhova** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @elenavolokhova[/z](https://www.0crat.com/u/elenavolokhova)

**Pull request merged**